### PR TITLE
Fix deprecation warning

### PR DESCRIPTION
**Pull request by jugmac00**
Iterable should be imported from collections.abc instead of from
collections.

modified:   flask_wtf/file.py

### CODE REVIEW

**Comment by jugmac00** [comment]
Travis failure is unrelated to this pr - see https://github.com/lepture/flask-wtf/issues/390

File: flask_wtf/file.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
 import warnings
-from collections import Iterable
 
 from werkzeug.datastructures import FileStorage
 from wtforms import FileField as _FileField
 from wtforms.validators import DataRequired, StopValidation
 
 from ._compat import FlaskWTFDeprecationWarning
+from ._compat import PY2
+if PY2:
```
Comment (davidism) [inline flask_wtf/file.py]:
```
try:
    from collections import abc
except ImportError:
    import collections as abc
```

should go in `_compat` and that's what should be imported.

Comment (jugmac00) [reply flask_wtf/file.py]: Thank you - looks so much nicer!

**Comment by jugmac00** [comment]
@davidism Thank you for taking the time for the review.

I did not force-push in order to not mess with the code review. Please squash the commits when this is ready to be merged.

File: flask_wtf/_compat.py [hunk 2]
```diff
@@ -6,10 +6,12 @@
 if not PY2:
     text_type = str
     string_types = (str,)
+    from collections.abc import Iterable
```
Comment (davidism) [inline flask_wtf/_compat.py]: Please use the pattern I showed previously. I want to export `_compat.abc`, not `_compat.Iterable`.

Comment (jugmac00) [reply flask_wtf/_compat.py]: Sorry, time to quit for today. I hope it is fine now.

I am curious. Is there a technical or an aesthetical reason?

And why didn't you do something like this also?
```
if not PY2:
    import urllib.parse as urlparse
else:
    import urlparse
```

**Pull request merged**